Utility/Framework/SymfonyFramework.php

```diff
@@ -18,24 +18,6 @@ class SymfonyFramework
     /**
      * @return bool
      */
-    public static function hasDefinitionSharing()
```

Comment:
Not that anybody should have bound to this (or has had the time), but, should we deprecate this instead of flat-out removing?

Either way I think we ought to reference flag removal in the UPGRADE doc.